app/controllers/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+class UsersController < ApplicationController
+  def index
+    @teachers = User.where(is_teacher: true)
+  end
+
+  def show
+    @teacher = User.find(params[:id])
```

Comment:
should there be a 2nd condition here for params[:id] matching, but also is_teacher: true?
That way we don't ever show a student page